### PR TITLE
Replace print statements with logger

### LINKED ISSUE #4: Replace printed messages with logging module
Printing is fragile- we should log our messages with the `logging` module.

### PR DESCRIPTION
**Pull request by vinferrer**
Closes #4  .

Changes proposed in this pull request:

- change all the print messages by the ``LRG.info`` or ``LRG.warning``
- add the quiet or Debug mode when is necessary

### CODE REVIEW

File: aroma/_version.py [hunk 1]
```diff
@@ -86,20 +89,20 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
             if e.errno == errno.ENOENT:
                 continue
             if verbose:
-                print("unable to run %s" % dispcmd)
-                print(e)
+                LGR.info("unable to run %s" % dispcmd)
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
                LGR.info("Unable to run %s" % dispcmd)
```

File: aroma/_version.py [hunk 2]
```diff
@@ -86,20 +89,20 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
...
+                LGR.info(e)
             return None, None
     else:
         if verbose:
-            print("unable to find command, tried %s" % (commands,))
+            LGR.info("unable to find command, tried %s" % (commands,))
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
            LGR.info("Unable to find command, tried %s" % (commands,))
```

File: aroma/_version.py [hunk 3]
```diff
@@ -86,20 +89,20 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False,
...
         return None, None
     stdout = p.communicate()[0].strip()
     if sys.version_info[0] >= 3:
         stdout = stdout.decode()
     if p.returncode != 0:
         if verbose:
-            print("unable to run %s (error)" % dispcmd)
-            print("stdout was %s" % stdout)
+            LGR.info("unable to run %s (error)" % dispcmd)
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
            LGR.info("Unable to run %s (error)" % dispcmd)
```

File: aroma/_version.py [hunk 4]
```diff
@@ -175,7 +178,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
     refnames = keywords["refnames"].strip()
     if refnames.startswith("$Format"):
         if verbose:
-            print("keywords are unexpanded, not using")
+            LGR.info("keywords are unexpanded, not using")
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
            LGR.info("Keywords are unexpanded, not using")
```

File: aroma/_version.py [hunk 5]
```diff
@@ -192,22 +195,22 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # "stabilization", as well as "HEAD" and "master".
         tags = set([r for r in refs if re.search(r'\d', r)])
         if verbose:
-            print("discarding '%s', no digits" % ",".join(refs - tags))
+            LGR.info("discarding '%s', no digits" % ",".join(refs - tags))
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
            LGR.info("Discarding '%s', no digits" % ",".join(refs - tags))
```

File: aroma/_version.py [hunk 6]
```diff
@@ -192,22 +195,22 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
...
     if verbose:
-        print("likely tags: %s" % ",".join(sorted(tags)))
+        LGR.info("likely tags: %s" % ",".join(sorted(tags)))
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
        LGR.info("Likely tags: %s" % ",".join(sorted(tags)))
```

File: aroma/_version.py [hunk 7]
```diff
@@ -192,22 +195,22 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
...
     for ref in sorted(tags):
         # sorting will prefer e.g. "2.0" over "2.0rc1"
         if ref.startswith(tag_prefix):
             r = ref[len(tag_prefix):]
             if verbose:
-                print("picking %s" % r)
+                LGR.info("picking %s" % r)
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
                LGR.info("Picking %s" % r)
```

File: aroma/_version.py [hunk 8]
```diff
@@ -192,22 +195,22 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
...
             return {"version": r,
                     "full-revisionid": keywords["full"].strip(),
                     "dirty": False, "error": None,
                     "date": date}
     # no suitable tags, so version is "0+unknown", but full hex is still there
     if verbose:
-        print("no suitable tags, using unknown + full revision id")
+        LGR.info("no suitable tags, using unknown + full revision id")
```
Comment (eurunuela) [inline aroma/_version.py]:
```suggestion
        LGR.info("No suitable tags, using unknown + full revision id")
```

File: aroma/aroma.py [hunk 9]
```diff
@@ -22,17 +26,74 @@ def aroma_workflow(
...
     Parameters
     ----------
     in_feat
     """
-    print("\n------------------------ RUNNING ICA-AROMA ------------------------")
-    print("-------- 'ICA-based Automatic Removal Of Motion Artifacts' --------\n")
+    LGR.info("\n------------------------ "
+             "RUNNING ICA-AROMA ------------------------")
+    LGR.info("-------- "
+             "'ICA-based Automatic Removal Of Motion Artifacts' --------\n")
```
Comment (eurunuela) [inline aroma/aroma.py]: Do we really want this? I mean, anyone using this package will know they're using ICA-AROMA. I'd remove these lines.

**Comment by vinferrer** [comment]
Eyy folks, i think this is ready for review

**Review by eurunuela — COMMENTED** [review]
I think this is great Vicente! I'd keep the same format on every logged line though. For example, starting lines with CAPS.

File: aroma/aroma.py (hunk 9 above)
Comment (vinferrer) [reply aroma/aroma.py]: They were already in the code

**Review by eurunuela — APPROVED** [review]
Thank you! LGTM!

**Pull request merged**